tests/dicom/test_dicom_utils.py

```
@@ -1,16 +1,14 @@
-from rich.tree import Tree
-import pytest
-from pathlib import Path
-from tempfile import TemporaryDirectory
-from imgtools.dicom.utils import find_dicoms
 
 # from imgtools.sort import lookup_tag, similar_tags, tag_exists
 # from imgtools.sort.exceptions import InvalidPatternError, SorterBaseError
 # from imgtools.sort.sorter_base import SorterBase
-from imgtools.dicom import lookup_tag, similar_tags, tag_exists, find_dicoms
+from imgtools.dicom import find_dicoms, lookup_tag, similar_tags, tag_exists
+from imgtools.dicom.utils import find_dicoms
```

Ruff (0.8.2)

Redefinition of unused `find_dicoms` from line 9

(F811)